src/translator.h

```diff
@@ -143,7 +143,7 @@ class Translator
     virtual QCString getLanguageString() = 0;
 
     /** add punctuation at the end of a brief description when needed and supported by the language */
-    virtual const inline bool needsPunctuation() {return true;}
+    virtual inline bool needsPunctuation() {return true;}
```

Comment:
`inline` is also not useful for virtual functions especially when called via the base class, adding `const` after `()` could be done as this method will not modify the state of the object.

Reply:
I've found some more problems, so I'm looking into the issue.